packages/app/src/cli/utilities/app/workspaces-warning.ts

```diff
@@ -0,0 +1,29 @@
+import {PackageManager} from '@shopify/cli-kit/node/node-package-manager'
+import {formatPackageManagerCommand} from '@shopify/cli-kit/node/output'
+import {renderWarning} from '@shopify/cli-kit/node/ui'
+
+const links = {
+  npm: 'https://docs.npmjs.com/cli/v7/using-npm/workspaces',
+  yarn: 'https://classic.yarnpkg.com/lang/en/docs/workspaces/',
+  pnpm: 'https://pnpm.io/workspaces',
+}
+
+export function showWorkspaceWarning(packageManager: PackageManager) {
+  renderWarning({
+    headline: 'Dependency installation behavior soon to change',
+    body: [
+      'The CLI will soon stop installing dependencies in project subdirectories by default. To maintain the same behavior, please transition over to using workspaces by running',
+      {command: formatPackageManagerCommand(packageManager, 'app migrate workspaces')},
```

Comment:
This doesn't exist, we should consider not showing this alert if there is no way to convert your project to use workspaces (yet)